app/build.gradle

```diff
@@ -77,6 +77,9 @@ dependencies {
     implementation "com.google.android.gms:play-services-oss-licenses:$rootProject.playServicesVersion"
     implementation "com.isseiaoki:simplecropview:$rootProject.cropviewVersion"
 
+    // Multidex
```

Comment:
@m-sameer Remove it. PR #909  will be merged then you can pull that code and update your PR.